src/__tests__/linter.test.ts

```diff
@@ -418,6 +429,7 @@ describe('linter', () => {
 
   test('should not run any rule with defined formats if some formats are registered but document format could not be associated', async () => {
     spectral.registerFormat('foo-bar', obj => typeof obj === 'object' && obj !== null && 'foo-bar' in obj);
+    spectral.registerFormat('markdown', () => false);
```

Comment:
This doesnt seem to be doing much?

Reply:
Actually, no, you are right. I'll remove it